ubuntudesign/cloud-init
# Media unit fixup

## Done

Fixed up media unit so that images and text stack on smaller screen until there is enough space for them to sit side-by-side.

## QA

Run code, squeeze screen to less that 468px wide and observe that images and text of "The Team" section of the page should stack nicely.
